plugins/Historic_Wayside_cross_without_material.py

```diff
@@ -0,0 +1,56 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights: Morray 2020                                               ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from plugins.Plugin import Plugin
+
+
+class Historic_Wayside_cross_without_material(Plugin):
+
+    only_for = ["DE", "AT", "CH"]
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[800] = self.def_class(item = 9200, level = 1, tags = ['historic', 'fix:survey'],

```

Comment:
```suggestion
        self.errors[303241] = self.def_class(item = 3032, level = 1, tags = ['historic', 'fix:survey'],
```

Item 3020 is relation multiple tags.

Item in 9xxx cannot be reused, there are generated from JOSM Mapcss Validator.